<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>298</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>矩阵相加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_minus">
       <property name="text">
        <string>矩阵相减</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_transpose">
       <property name="text">
        <string>矩阵转置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>70</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>行 ×列</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_row">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_col">
       <property name="maxLength">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_creat">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_information">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请先输入第一个矩阵大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>矩阵1：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>矩阵2：</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>100</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>491</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string>--------------------------------------------------------------</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
